printer_mods/README.md

```diff
@@ -92,6 +92,7 @@ like so:
 |consibonsi | [Y-Mod SW 9mm](./consibonsi/Y-Mod_SW_9mm)| 9mm Y belt mod for SW | :x: |:x: |:x: |:heavy_check_mark:|
 |csch0 | [SW Fan Mount No Tape](./csch0/SW_FanMount_No_Tape)| 60x20 Fan Mount for SW without tape| :x: |:x: |:x: |:heavy_check_mark:|
 | d3nny | [Filtered Power Plug with RND_165-00057](./d3nny/mains_plug_panel_for_RND_165-00057) | Filtered Power Plug for RND 165-00057 and CAT6-Plug | :x: | :x: | :heavy_check_mark: |:x:|
+|D4rkCelerity| [Raspberry Pi Zero Bracket](./D4rkCelerity/Raspberry_Pi_Zero_Bracket/) | Modified version of Trident RPi bracket to fit the smaller RPi Zero | :x: | :grey_question: | :grey_question: | :x: | :heavy_check_mark: |
```

Comment:
if you want the check the v0 it should be fine since it can be attached with VHB. You can also check the v2 and v1.8 columns, since they both use DIN rail mounts.